cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java

```diff
@@ -34,15 +33,17 @@
 
 public class IntercomMetricCollector implements MetricCollector {
     private final Logger logger = LoggerFactory.getLogger(getClass());
-    private MetricCustomAttributeToIntercomCustomAttributeMapper customAttributeMapper;
+    private MetricCustomAttributeToIntercomCustomAttributeMapper metricToIntercomCustomAttributeMapper;
+    private final IntercomCustomAttributeToMetricCustomAttributeMapper intercomToMetricCustomAttributeMapper;
```

Comment:
@clicktravel-chris The non-final one (Metric->Intercom) was there already, with a setter, so I haven't touched it. But I don't know the reason for the setter as it is already instantiated in the constructor. So I have not created a setter for the new one (Intercom->Metric). Could do with some confirmation if actually a setter is needed or not.